recipes/pepquery/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set name = "pepquery" %}
+{% set version = "1.1" %}
+{% set sha256 = "c819b0405392f13c7a7bfd6df11c3a639a3fe68c5649a136167aa38f40107358" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: generic
+
+source:
+  url: https://github.com/jraysajulga/PepQuery/archive/v{{ version }}.zip
+  sha256: {{ sha256 }}
+
+requirements:
+  host:
+    - openjdk
+  run:
+    - openjdk
+
+test:
+  commands:
+    - pepquery -printPTM >/dev/null; rm -rf logs/
```

Comment:
you do not need to remove the logs in a test ... tests will not end up in a package

Reply:
I had permission issues with the log files generated through the test command through CircleCI building and testing:

`File "/home/circleci/project/miniconda/lib/python3.6/shutil.py", line 436, in _rmtree_safe_fd
    os.unlink(name, dir_fd=topfd)
PermissionError: [Errno 13] Permission denied: 'log4j.properties'
Exited with code 1`

Reply:
Builds and tests fine on my local instance though...

Reply:
ok, wired, can you maybe add a comment about this to the recipe

Reply:
Sure. Not sure if I added a comment with best practices, so feel free to check.